Lesson2 arrays

VladimirSinitsyn opened the pull request:
Реализуйте метод, принимающий в качестве аргументов целое число и строку, и печатающий в консоль строку указанное количество раз
-Реализуйте метод, принимающий в качестве аргумента целочисленный массив, суммирующий все элементы, значение которых больше 5, и печатающий полученную сумму в консоль.
-Реализуйте метод, принимающий в качестве аргументов целое число и ссылку на целочисленный массив, метод должен заполниться каждую ячейку массива указанным числом.
-Реализуйте метод, принимающий в качестве аргументов целое число и ссылку на целочисленный массив, увеличивающий каждый элемент которого на указанное число.
-Реализуйте метод, принимающий в качестве аргумента целочисленный массив, и печатающий в консоль сумма элементов какой из половин массива больше.

VladimirSinitsyn committed:
init main thread in github

VladimirSinitsyn committed:
Removed upper symbols in path naming and in packages

VladimirSinitsyn committed:
Removed upper symbols in path naming and in packages

VladimirSinitsyn committed:
Merge remote-tracking branch 'origin/lesson1' into lesson1

VladimirSinitsyn committed:
Removed upper symbols in path naming and in packages

VladimirSinitsyn committed:
Changed lesson1 package to Lesson1
Removed "else if" construction for increment in 5th task to "else"

VladimirSinitsyn committed:
Merge branch 'lesson1' of https://github.com/VladimirSinitsyn/ru_otus_java_basic_12_2023

VladimirSinitsyn committed:
add from main with next removing

VladimirSinitsyn committed:
Merge branch 'lesson1' of https://github.com/VladimirSinitsyn/ru_otus_java_basic_12_2023

VladimirSinitsyn committed:
Merge branch 'lesson1' of https://github.com/VladimirSinitsyn/ru_otus_java_basic_12_2023

VladimirSinitsyn committed:
Init new branch

VladimirSinitsyn committed:
//

VladimirSinitsyn committed:
SimpleConsoleCalc

VladimirSinitsyn committed:
HomeWork #2. One-Dimention Arrays
5 tasks

sproshchaev commented on src/main/java/ru/sinitsyn/java/basic/homeworks/homework2/Lesson2Arrays.java:
```diff
@@ -0,0 +1,150 @@
+package Ru.Sinitsyn.Java.Basic.Homeworks.homework2;
```
Имя пакета пишется строчными буквами

VladimirSinitsyn replied:
Переименовал пакет практически полностью через Refactor-Rename.
Не поменялось только одно слово Basic.
Видимо глюк. Попробую позже еще раз

sproshchaev commented on src/main/java/ru/sinitsyn/java/basic/homeworks/homework2/Lesson2Arrays.java:
```diff
@@ -0,0 +1,150 @@
+
+    public static void main(String[] args) {
+        /*Задача1
+         * Реализуйте метод, принимающий в качестве аргументов целое число и строку, и печатающий в консоль строку указанное количество раз*/
```
Здесь и далее: комментарии необходимо удалять перед выполнением pull request. Если какую-то информацию относительно работы метода хочется оставить, то ее можно оформить в виде javadoc (см. пример https://java-online.ru/java-javadoc.xhtml)

VladimirSinitsyn replied:
комментарии удалены.

sproshchaev commented on src/main/java/ru/sinitsyn/java/basic/homeworks/homework2/Lesson2Arrays.java:
```diff
@@ -0,0 +1,150 @@
+    /*Задача1
+     * Реализуйте метод, принимающий в качестве аргументов целое число и строку, и печатающий в консоль строку указанное количество раз*/
+    public static void task1UsageFor() {
+        //инициализируем строковую переменную a, которую в последующем будем выводить в консоль
```
Здесь и далее: комментарии в методах необходимо удлаять перед выполнением pull request

sproshchaev commented on src/main/java/ru/sinitsyn/java/basic/homeworks/homework2/Lesson2Arrays.java:
```diff
@@ -0,0 +1,150 @@
+            при этом String пишется с большой буквы, а значение переменной в "" */
+            System.out.println(a);
+        }
+
```
Пустые строки внутри метода класса рекомендуется удалять, чтобы быть более компактным для удобства его чтения

sproshchaev commented on src/main/java/ru/sinitsyn/java/basic/homeworks/homework2/Lesson2Arrays.java:
```diff
@@ -0,0 +1,150 @@
+
+    /*Задача1
+     * Реализуйте метод, принимающий в качестве аргументов целое число и строку, и печатающий в консоль строку указанное количество раз*/
+    public static void task1UsageFor() {
```
Здесь и далее: если написано, что метод должен принимать что-то в качестве аргументов - то в сигнатуре метода это что-то должно быть, например:

public static void task1UsageFor(int num, String str)

sproshchaev commented on src/main/java/ru/sinitsyn/java/basic/homeworks/homework2/Lesson2Arrays.java:
```diff
@@ -0,0 +1,150 @@
+        Scanner scanner = new Scanner(System.in);
+        //Инициализируем переменную для числа, которым будет заполнен первый элемент массива
+        System.out.println("Введите целое число, которым будет заполнен первый элемент массива");
+        int numberForFillingArray = scanner.nextInt();
```
Сделайте, чтобы число и массив передавались в качестве аргумента, а запрос числа перенесите в метод main

VladimirSinitsyn committed:
Remove comments,
Changed 1st task. Added arguments in method.
Applied small requested changes, like deleting blank strings.
